src/language-css/printer-postcss.js

```diff
@@ -115,12 +115,20 @@ function genericPrint(path, options, print) {
         options.locStart(node),
         options.locEnd(node)
       );
+
       const rawText = node.raws.text || node.text;
       // Workaround a bug where the location is off.
       // https://github.com/postcss/postcss-scss/issues/63
       if (text.indexOf(rawText) === -1) {
-        if (node.raws.inline) {
-          return concat(["// ", rawText]);
+        if (node.raws.inline || node.inline) {
+          const needBreakAfter = !(
+            node.source.input.css.split(/[\r\n]/)[node.source.end.line] || ""
+          ).trim();
+          return concat([
+            "//",
+            node.raws.left + rawText,
+            needBreakAfter ? line : ""
+          ]);
```

Comment:
I think we can simplify that, it was workaround for less parsers bugs, so maybe they were fixed

Reply:
Will check this

Reply:
Seems hard to fix, maybe fix this later